mlir-tensorrt/compiler/lib/Compiler/TensorRTToExecutable/Passes.cpp

```diff
@@ -118,7 +132,7 @@ outlineOp(RewriterBase &rewriter, tensorrt::TensorRTModuleOp trtModule,
       makeRegionIsolatedFromAbove(rewriter, inlineGroupOp.getRegion());
 
   // Create the outlined function
-  FailureOr<FunctionOpInterface> func = createOutlinedFunc(
+  FailureOr<FunctionOpInterface> func = createOutlinedFunc(outerModule,
```

Comment:
The function which contains the cluster can be found via `cluster.back().getParentOfType<func::FuncOp>()`.